add more details in the initial request (#39)
tradingview-pine-seeds/docs: repo.md

Specify your GitHub username and the desired repository suffix.

> Note that the maximum number of characters in the suffix is 16 and in the whole repository name is 64.

Reviewer: ```suggestion
Note that the maximum number of characters allowed for the suffix is 16, and the overall repository name should not exceed 64 characters.
```

> The repository name will be `seed_<your_github_username>_<suffix_you_provided>`.

Reviewer: ```suggestion
The repository name format should be `seed_<your_github_username>_<suffix_you_provided>`.
```

> In your message please also describe how you intend to use Pine Seeds and how did you find out about it.

Reviewer: ```suggestion

Additionally, describe in your email how you plan to use Pine Seeds and how you came across it.
This information will be valuable to us in our efforts to enhance the product.
```
